### PR TITLE
Phrase

### LINKED ISSUE #4: Create Migrations for Phrases
- Add Attributes:
id
expression
icon
numbers of uses

### PR DESCRIPTION
**Pull request by sagemlee**
### Description
Added Phrase table migration, model, and query test. Not sure that the front end will ever use this query phrase by id, but I just wanted to make sure it was working.
Also changed user timestamps back to snake case.
Also added some gems that should  fix the problems we were having with deployment.
Removed some unnecessary commented lines.
### Type of change
 - [x] Bug fix (non-breaking change which fixes an issue)
 - [x] Refactor (non-breaking change which maintains existing functionality)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -17,6 +17,10 @@ gem 'figaro'
 
 gem 'graphql'
 
+gem 'sass-rails'
```
Comment (sagemlee) [inline Gemfile:20]: These are the gems that allow us to use graphiql without the sprocket line we previously had in our application.rb.

File: config/application.rb [hunk 2]
```diff
@@ -10,7 +10,7 @@
 require "action_mailer/railtie"
 require "action_view/railtie"
 require "action_cable/engine"
- require "sprockets/railtie"
+# require "sprockets/railtie"
```
Comment (sagemlee) [inline config/application.rb:13]: This is the line that was causing our heroku issues, without it, things should run smoothly.

File: Gemfile (hunk 1 above)
Comment (EricLarson2020) [reply Gemfile:20]: Nice work, so does that mean we re-comment in that line?

File: app/graphql/types/phrase_type.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+module Types
```
Comment (EricLarson2020) [inline app/graphql/types/phrase_type.rb:1]: Nice Work here!

File: config/application.rb (hunk 2 above)
Comment (EricLarson2020) [reply config/application.rb:13]: Awesome! Great work catching that!

File: db/migrate/20200907022855_change_timestamps_snake_case.rb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+class ChangeTimestampsSnakeCase < ActiveRecord::Migration[5.2]
+    def change
+    rename_column :users, :createdAt, :created_at
```
Comment (EricLarson2020) [inline db/migrate/20200907022855_change_timestamps_snake_case.rb:3]: Thanks for changing that back

**Review by EricLarson2020 — APPROVED** [review]

File: Gemfile (hunk 1 above)
Comment (sagemlee) [reply Gemfile:20]: You should be able to!

File: config/application.rb (hunk 2 above)
Comment (cgaddis36) [reply config/application.rb:13]: yes nice work here!

**Pull request merged**